fastapi_route_logger_middleware/__init__.py

```diff
@@ -27,9 +28,18 @@ async def dispatch(self, request: Request, call_next: Callable) -> Response:
 
     def _should_route_be_skipped(self, request: Request) -> bool:
         return any(
-            [path for path in self._skip_routes if request.url.path.startswith(path)]
+            filter(self.__skip_route_match_filter(request.url.path), self._skip_routes)
```

Comment:
Why switch to filter and the complexity of a function that returns a function?

Reply:
I chose this direction because the logic is now more complex than should fit as a one-liner.